src/styles/base.scss

```diff
@@ -4,11 +4,27 @@
 @import './mixins.scss';
 @import './grid.scss';
 @import './font.scss';
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500,700');
 
 body{
-  font-family: 'Open Sans';
+  font-family: 'Roboto', sans-serif;
```

Comment:
Is there any specific reason that you have changed the font? @live-wire

Reply:
In @shivaniprakash95's designs, we have this new font. This was decided upon apparently.

Reply:
Let's stick to Roboto then @live-wire